scalardl-test/src/main/java/scalardl/transfer/TransferChecker.java

```diff
@@ -27,12 +27,24 @@
 
 public class TransferChecker extends PostProcessor {
 
+  private final long sleepTimeMilliseconds;
+
   public TransferChecker(Config config) {
     super(config);
+    this.sleepTimeMilliseconds =  config.getUserLong("test_config", "sleep_time_milli_sec", 0L);
   }
 
   @Override
   public void execute() {
+
+    try {
+      Thread.sleep(sleepTimeMilliseconds);
```

Comment:
Rather than adding a sleep, I think we should make the retry options configurable and adjust them. It looks like we can adjust `initialIntervalMillis` and `multiplier` for `ExponentialBackoff` and `maxAttempts`:
https://github.com/scalar-labs/kelpie-test/blob/c7279de7f53cb75a713befc13721874282cda51d/scalardl-test/src/main/java/scalardl/Common.java#L77-L80

What do you think?

Reply:
Thank you for the suggestion. I think this would be better compared to adding sleep.
I will change MAX_RETRIES and SLEEP_BASE_MILLIS to configurable and test by adjusting this values to confirm whether the issue is occurring or not. 
I will update changes based on test results.